src/components/common/Picker/styles.js

```diff
@@ -0,0 +1,67 @@
+import { BLACK, ERROR } from 'constants/colors';
+import { PRIMARY_FONT_SEMI_BOLD } from 'constants/fonts';
+import { StyleSheet } from 'react-native';
+
+export const pickerSelectStyles = isError =>
+  StyleSheet.create({
+    inputIOS: {
```

Comment:
it's this a valid style name?

Reply:
I don't understand. It's the same name use in the example docs https://snack.expo.dev/@lfkwtz/react-native-picker-select . Do you suggest another name like **stylesPicker** ?

Reply:
It just looks weird because you don't have to do a `Platform.select`, so it looks like you declared both `inputIOS` and `inputAndroid` but you never used them. It seems like the library does that by itself.

It's explicitly mentioned in [`react-native-picker-select`'s README](https://github.com/lawnstarter/react-native-picker-select#styling), so it's all good!

Reply:
It just looks weird because you don't have to do a `Platform.select`, so it looks like you declared both `inputIOS` and `inputAndroid` but you never used them. It seems like the library does that by itself.

It's explicitly mentioned in [`react-native-picker-select`'s README](https://github.com/lawnstarter/react-native-picker-select#styling), so it's all good!